src/test/java/co/rsk/federate/signing/hsm/client/HSMClientVersion2BTCTest.java

```diff
@@ -466,6 +465,35 @@ public void updateAncestorBlock_hsm_version_3() throws HSMClientException, JsonR
         Assert.assertFalse(capturedArguments.get(1).has("brothers"));
     }
 
+    @Test
+    public void updateAncestorBlock_hsm_version_4() throws HSMClientException, JsonRpcException {
```

Comment:
It's pretty much the same test repeated for version 2, 3 and 4 right? Wouldn't it make sense to me it into a function and pass the different versions as param?

Reply:
Yes It Does, Refactored